model/channel_shared.go

```diff
@@ -113,7 +113,7 @@ type SharedChannelRemote struct {
 	IsInviteAccepted  bool   `json:"is_invite_accepted"`
 	IsInviteConfirmed bool   `json:"is_invite_confirmed"`
 	RemoteId          string `json:"remote_id"`
-	LastSyncAt        int64  `json:"last_sync_at"`
+	NextSyncAt        int64  `json:"next_sync_at"`
```

Comment:
Can you explain the reasons behind the naming change? I am asking because I can't see it from these changes. Was it incorrect before?

Reply:
If was correct before in that it accurately described what it represented.  The new algorithm ensures all posts with matching UpdateAt values get put into a single sync batch, so the next query can always be the UpdateAt from the last post in the previous batch plus 1 millisecond.  So now it no longer describes the last UpdateAt sync'd - it is where we should sync from next.

This will be simplified once we have a monotonic flakeId for UpdateAt.